pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -407,207 +331,80 @@ func (exp *flowExporter) sendTemplateSet(isIPv6 bool) (int, error) {
 	return sentBytes, nil
 }
 
-func (exp *flowExporter) addRecordToSet(record flowexporter.FlowRecord) error {
-	// Iterate over all infoElements in the list
+func (exp *flowExporter) addConnToSet(conn flowexporter.Connection) error {
```

Comment:
there is no reason for this function to take the connection by value? IIRC, `flowexporter.Connection` is a large struct.